libcontainer/seccomp/patchbpf/enosys_linux.go

```diff
@@ -641,8 +646,13 @@ func filterFlags(config *configs.Seccomp, filter *libseccomp.ScmpFilter) (flags
 			flags |= uint(C.C_FILTER_FLAG_LOG)
 		}
 	}
-
-	// TODO: Support seccomp flags not yet added to libseccomp-golang...
```

Comment:
is this the only flag not supported by libseccomp-golang? If not, is this comment still relevant?

Reply:
Currently, libseccomp-golang is on par with the runtime-spec, thus I removed this comment.

Reply:
Currently correct, but I'd like support for SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV (Linux 5.19, https://github.com/torvalds/linux/commit/c2aa2dfef243).

https://github.com/opencontainers/runtime-spec/pull/1161

Not yet in libseccomp: https://github.com/seccomp/libseccomp/issues/387

Reply:
Right, but I guess we'll support it in libseccomp and libseccomp-golang first, when here in runc.